app/Console/Commands/CacheFees.php

```diff
@@ -43,12 +43,12 @@ public function handle(FeeCache $cache): void
             $cache->setHistorical($period, HistoricalAggregateFactory::make($period)->aggregate());
         });
 
-        collect(Forms::getTransactionOptions())->except('all')->keys()->each(function ($type) use ($cache): void {
+        collect(Forms::getTransactionOptions())->except(['all'])->keys()->each(function ($type) use ($cache): void {
             preg_match('/^[a-z]+(\d+)$/', self::LAST_20, $match);
 
             $result = (new LastFeeAggregate())
                 ->setLimit((int) $match[1])
-                ->setType($type ?? '')
```

Comment:
`$type` is already non-nullable string, so null-coalescing operator `??` has no effect.

Reply:
could also change it to `->each(function (string $type)` to make it explicit that it's not null 🤔